source/fundamentals/crud/read-operations/geo.txt

```diff
@@ -85,6 +85,23 @@ To learn more about the shapes you can use in MongoDB, see the
 
 .. external resource
 
+Insert a Document Containing GeoJSON Data
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To insert a document that stores GeoJSON data, specify a document that contains a GeoJSON
```

Comment:
S: to me, "specify a document" is a little funny on its own like this. looks like you used "create a document" in the next section; i think that works better

```suggestion
To insert a document that stores GeoJSON data, create a document that contains a GeoJSON
```